### DISCUSSION TITLE
Storing hosting.properties, docker-compose.yaml & envoy.yaml in root home?

### DISCUSSION THREAD

**Discussion by anuone** [Budibase/budibase#4114] [Help]
Hey, i am relatively new to Budibase and have a question regarding platform security.

I followed the docs on the docker-compose setup with Caddy and got everything working so far. Yet are there any specifications to where one should store the docker-compose.yaml, envoy.yaml and hosting.properties files? Especially the latter one, as it contains the environment variables for the plattform. For now i actually just downloaded it all to the root users home folder, the one that Digital Ocean sends you after sshing into the server. i changed the variables in hosting.properties and ran the up command. Now i am wondering whether this has any implications for the security of the platform?

Thanks guys, I am really impressed by your project!

**Comment by mjashanks** [comment]
Hey @anuone - welcome,  and thanks!

It's really up to you where you want to store it, and up to you to secure that droplet and folder. We don't make decisions about the admin and security of your DO server - so it's difficult to give you advice here.

I'd guess you'd need to think about 
- Who has access to that login?
- What other logins have access to that server? Do they have root access? i.e. can they access other users' home directories?
- Who has access to your DO account (and could reset your passwords etc)?

I am far from a network administrator by the way, so this advice includes the most basic questions! 

Ultimately, when you choose to self-host (anything, not just Budibase), it's your responsibility to understand your network security.
